internal/usecase/repository/repo.go

```diff
@@ -0,0 +1,43 @@
+package repository
+
+import (
+	"address-book/internal/entity"
+)
+
+type Repository []*entity.AddressField
+
+func NewRepository() *Repository {
+	return new(Repository)
+}
+
+func (r Repository) FindField(param string) (index int, ok bool) {
```

Comment:
What if we find more than one item? For example, we are going to find users by name using wildcard `*`, and we have more than one users in our address-book